### PR TITLE
Update golang deps

### LINKED ISSUE #751: Make update-repos work with multiple go.mod
As it's not really supported by gazelle, we can look into how kubernetes solves the problem

### PR DESCRIPTION
**Pull request by chrismatix**
resolves #751 

TODO:
- [x] write latest version of each library back to the go.mod files and run `go get` on each of them

### CODE REVIEW

File: backend/sources/facebook/events-router/src/main/java/co/airy/core/sources/facebook/MessageParser.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ String getSourceConversationId(final JsonNode webhookMessaging) throws NullPoint
         } else if (appId != null && !appId.equals(this.facebookAppId)) {
             senderType = SenderType.SOURCE_USER;
             senderId = appId;
-        } else if(isEcho && appId == null) {
+        } else if(appId == null) {
```
Comment (paulodiniz) [inline backend/sources/facebook/events-router/src/main/java/co/airy/core/sources/facebook/MessageParser.java:56]: This brings back bad memories

File: tools/update-deps/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Update Golang deps tool
```
Comment (chrismatix) [inline tools/update-deps/README.md:1]: See here for a description of the PR content

**Review by ljupcovangelski — APPROVED** [review]
Awesome :tada:

**Comment by pascal-airy** [comment]
I need to see this run with my own eyes before I believe it 😮

**Review by pascal-airy — APPROVED** [review]
image

**Review by lucapette — APPROVED** [review]

**Pull request merged**